templates/back-office/transfer-packing-slip.hypr

```diff
@@ -204,32 +352,101 @@
-            <th>Product Code</th>
-            <th>Product Name</th>
-            <th>Quantity</th>
-            <th>Weight</th>
+            <th>{{ labels.productCodePackingSlip }}</th>
+            <th>{{ labels.productNamePackingSlip }}</th>
+            <th>{{ labels.quantityPackingSlip }}</th>
+            <th>{{ labels.weightPackingSlip }}</th>
         </tr></thead>
+        {% endif %}
         <tbody>
         {% for item in model.items %} <!-- model represents the shipment -->
-           <tr>
-                <td>
-                    {{ item.productCode }}
-                </td>
-                <td>
-                    <div class="product-name">{{ item.name }}</div>
-                </td>
-                <td>
-                    {{ item.quantity }}
-                </td>
-                <td>
-                    {{ item.weight }} {{ item.weightUnit }}
-                </td>
-            </tr>
+            {% if (not item.originalLineId) %}
+                {% if (item.quantity > 0) %}
+                <tr>
+                    <td>
+                        {{ item.productCode }}
+                    </td>
+                    <td>
+                        <div class="product-name">{{ item.name }}</div>
+                    </td>
+                    <td>
+                        {{ item.quantity }}
+                    </td>
+                    <td>
+                        {{ item.weight }} {{ item.weightUnit }}
+                    </td>
+                </tr>
+                {% endif %}
+            {% endif %}
         {% endfor %}
         </tbody>
     </table>
+    {% if model.substitutedItems.length > 0 %}
+        {% if showSubstituteItemHeaderInElse %}
+            <table class="grid no-border">
+                <tr>
+                    <td class="no-left-padding"><b>{{ labels.substituteBy }}:</b><br></td>
+                </tr>
+            </table>
+        {% endif %}
+    <table class="grid substitute-table">
+        {% if showSubstituteItemHeaderInElse %}
+        <thead>
+            <tr>
+                <th class="break-word">{{ labels.productSubstituted }}</th>
+                <th class="break-word">{{ labels.substituteProductCode }}</th>
+                <th class="break-word">{{ labels.substituteProductName }}</th>
+                <th class="break-word">{{ labels.quantitySubstituted }}</th>
+                <th class="break-word">{{ labels.weightSubstituted }}</th>
+            </tr>
+        </thead>
+        {% endif %}
+        <tbody>
+            {% for substituteItem in model.items %}
+                {% if substituteItem.originalLineId && substituteItem.quantity > 0 %}
+                <tr>
+                    <td>
+                        {% for item in model.items %}
+                        {% if substituteItem.originalLineId == item.lineId %}
+                        <div class="product-name">{{ item.name }}</div>
+                        {% endif %}
+                        {% endfor %}
```

Comment:
Are you not formatting the script tags for a reason; not wanting to indent the div tag further?

For example:
```
                        {% for item in model.items %}
                            {% if substituteItem.originalLineId == item.lineId %}
                                <div class="product-name">{{ item.name }}</div>
                            {% endif %}
                        {% endfor %}
`